index.html

```diff
@@ -261,88 +239,61 @@ <h2>Credential</h2>
-              <aside
-                class="example"
-                title="verifiable credential (encoded as JWT)"
-              >
-                <pre>
-eyJraWQiOiJ1cm46ZXhhbXBsZTppc3N1ZXIja2V5LTAiLCJhbGciOiJFUzI1NiIsInR5cCI6InZjK2xkK2p3dCIsImN0eSI6InZjK2xkK2pzb24ifQ.eyJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvbnMvY3JlZGVudGlhbHMvdjIiXSwiaWQiOiJodHRwOi8vZXhhbXBsZS5lZHUvY3JlZGVudGlhbHMvMzczMiIsInR5cGUiOlsiVmVyaWZpYWJsZUNyZWRlbnRpYWwiLCJVbml2ZXJzaXR5RGVncmVlQ3JlZGVudGlhbCJdLCJpc3N1ZXIiOiJodHRwczovL2V4YW1wbGUuZWR1L2lzc3VlcnMvMTQiLCJpc3N1YW5jZURhdGUiOiIyMDEwLTAxLTAxVDE5OjIzOjI0WiIsImNyZWRlbnRpYWxTdWJqZWN0Ijp7ImlkIjoiZGlkOmV4YW1wbGU6MTIzIiwiZGVncmVlIjp7InR5cGUiOiJCYWNoZWxvckRlZ3JlZSIsIm5hbWUiOiJCYWNoZWxvciBvZiBTY2llbmNlIGFuZCBBcnRzIn19fQ.pfbhgWlTUZA8WmoFbi8WEIUFyC_lSQaAswoW87D1YeimdWZLq4MiJ3o-CmTkvkEQFhffvRiCzmkhxjS_R_RdOw
-                </pre>
-              </aside>
-            </section>
-        </section>
+          </pre>
       </section>
         <section>
-          <h2>Securing JSON-LD VPs with JOSE</h2>
+          <h2>Securing JSON-LD Verifiable Presentations with JOSE</h2>
           <p>
             This section details how to secure verifiable presentations with the type 
             <code>application/vp+ld+json</code> with JOSE.
           </p>
           <p>[[rfc7515]] MAY be used to secure this media type.</p>
-          <p>When using this approach, the <code>typ</code> MUST be <code>vp+ld+jwt</code></p>
-          <p>When using this approach, the <code>cty</code> MUST be <code>vp+ld+json</code></p>
+          <p>The <code>typ</code> parameter MUST be <code>vp+ld+jwt</code></p>
+          <p>When present, the <code>cty</code> parameter MUST be <code>vp+ld+json</code></p>
           <p>
             See <a data-cite="rfc7515#section-4.1.10">Common JOSE Header Parameters</a> 
             for additional details regarding usage of <code>typ</code> and <code>cty</code>.
           </p>
-          <section>
-            <h2>Examples</h2>
-            <section>
-              <h2>Presentation Header</h2>
-              <aside
-                class="example"
-                title="verifiable presentation header (decoded JWT header)"
-              >
-                <pre class="example">
+          <pre class="example vc-jwt" data-vp-alg="none" title="The minimal unsecured verifiable presentation">
 {
-  "kid": "urn:example:holder#key-0",
-  "alg": "ES256",
-  "typ": "vp+ld+jwt",
-  "cty": "vp+ld+json"
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2"
+  ],
+  "type": ["VerifiablePresentation"]
 }
-                </pre>
-              </aside>
-            </section>
-            <section>
-              <h2>Presentation</h2>
-              <aside
-                class="example"
-                title="verifiable presentation (decoded JWT claimset)"
-              >
-                <pre class="example">
+          </pre>
+          <pre class="example vc-jwt" title="The minimal secured verifiable presentation">
```

Comment:
Here and elsewhere. Tabs are being rendered with the `Verifiable Credential` title, even though it is a `Verifiable Presentation`. Should this be updated? I'm guessing it's the script? 

<img width="577" alt="image" src="https://github.com/w3c/vc-jwt/assets/1139998/19ff2d5d-eabb-48a7-aff8-402147af2724">

Reply:
great catch, its here https://github.com/transmute-industries/respec-vc-jwt/blob/main/index.js#L323

Reply:
I filed here: https://github.com/transmute-industries/respec-vc-jwt/issues/1